spec/plans/results.fmf

```diff
@@ -141,6 +141,9 @@ description: |
           check:
             ...
 
+          # String, path to /data directory storing possible test artifacts
+          data-path: path/to/test/data
```

Comment:
We do have `data-path` for the whole test, `$TMT_TEST_DATA`, but should we have one for subresults as well?